website/.env.dev

```diff
@@ -0,0 +1,2 @@
+VITE_API_QUEUE_URL=http://localhost:9090/api/queue
+VITE_API_RECENT_URL=http://localhost:9090/api/recent
```

Comment:
I think it would be better to have only the domain + the port, i.e.:

```env
VITE_API=http://localhost:9090/api/
```

And then you can append the route in your code. That way we don't explicitly list all routes in a single file.